recipe/cakephp.php

```diff
@@ -44,6 +44,7 @@
  * Main task
  */
 task('deploy', [
+    'check_for_known_bugs',
```

Comment:
maybe instead of changing every single recipe add `before('deploy', 'check_for_bugs')`?

Reply:
Nice idea. I like it. Will do my best to change asap.